libs/contrib/Data/String/Parser.idr

```diff
@@ -153,3 +153,33 @@ option def p = p <|> pure def
 export
 optional : Monad m => ParseT m a -> ParseT m (Maybe a)
 optional p = (p >>= \res => pure $ Just res) <|> pure Nothing
+
+||| Discards the result of a parser
+export
+skip : Parser a -> Parser ()
+skip = map (const ())
```

Comment:
This is known as `ignore` and is available in the prelude:

```shell
Main> :t ignore
Prelude.ignore : Functor f => f a -> f ()
```

Reply:
Ah cool, didn't know that. Updated. I left the `skip` parser, but maybe I should delete it and just use `ignore` in the other parsers.